src/components/EditableInput/EditableInput.tsx

```diff
@@ -101,6 +128,7 @@ const EditableInput = forwardRef(
         }
 
         const enrichedEvent = event;
+        // @ts-ignore
```

Comment:
let's document the reason? if it's something that can be solved easily let's solve it.